Grbl_Esp32/i2s_ioexpander.cpp

```diff
@@ -185,11 +196,31 @@ static int i2s_stop() {
   return 0;
 }
 
+int i2s_ioexpander_start() {
+  i2s_start();
+  i2s_ioexpander_status = RUNNING;
+  return 0;
+}
+
+int i2s_ioexpander_stop() {
+  if (i2s_ioexpander_status == STOPPING) {
+    return -1;
+  }
+  i2s_ioexpander_status = STOPPING;
+  i2s_ioexpander_clear_buffer_counter = 0;
+  return 0;
+}
+
+int i2s_ioexpander_set_pulse_period(uint64_t period) {
+  i2s_ioexpander_pulse_period = period;
+  return 0;
+}
+
 //
 // I2S DMA Interrupts handler
 //
 static void IRAM_ATTR i2s_intr_handler_default(void *arg) {
-  int dummy;
```

Comment:
Could this be `lldesc_t *discard_desc` instead of dummy?  I think that would clarify the usage and meaning.

Reply:
I also think that variable names like "dummy" and "aaa" should be avoided. This code is the ESP-IDF codebase, but we should read and comment on the intent of the original code.